src/core/resources/_selectors/nfts.ts

```diff
@@ -16,76 +19,64 @@ export type NFTCollectionSectionData = {
 export const selectNfts = (data?: NFTInfiniteData) =>
   data?.pages?.map((page) => page.nfts).flat();
 
-export const sortSectionsAlphabetically = (
-  sections: NFTCollectionSectionData[],
+export const selectNftCollections = (
+  data?: NFTCollectionInfiniteData,
+  hiddenNfts?: Record<string, boolean>,
 ) => {
-  return sections.sort((a, b) => {
-    const aName = a.collection.name.toLowerCase();
-    const bName = b.collection.name.toLowerCase();
-    if (aName < bName) {
-      return -1;
-    }
-    if (aName > bName) {
-      return 1;
-    }
-    return 0;
-  });
-};
-
-export const sortSectionsByRecent = (sections: NFTCollectionSectionData[]) => {
-  return sections.sort((a, b) => {
-    const earliestDate = new Date(-8640000000000000);
-    const aCollectionAcquisition = a.lastCollectionAcquisition;
-    const bCollectionAcquisition = b.lastCollectionAcquisition;
-    const dateA = aCollectionAcquisition
-      ? new Date(aCollectionAcquisition)
-      : earliestDate;
-    const dateB = bCollectionAcquisition
-      ? new Date(bCollectionAcquisition)
-      : earliestDate;
-    return dateB.getTime() - dateA.getTime();
-  });
-};
-
-export const selectNftCollections = (data?: NFTInfiniteData) => {
-  const nfts = selectNfts(data);
-  const collections =
-    nfts?.reduce(
-      (collections, nft) => {
-        const currentCollectionId = nft.collection.collection_id;
-        if (currentCollectionId) {
-          const existingCollection = collections[currentCollectionId];
-          if (existingCollection) {
-            existingCollection.assets.push(nft);
-          } else {
-            collections[currentCollectionId] = {
-              assets: [nft],
-              collection: nft.collection,
-              lastCollectionAcquisition: nft.last_collection_acquisition,
-            };
-          }
-        }
-        return collections;
+  let idsOfCollectionsWithHiddenNfts: string[] = [];
+  return data?.pages
+    ?.map((page) => page.collections)
+    .flat()
+    .filter((c) => {
+      const collectionNftIds = c.nft_ids
+        .map((s) => s.split('.'))
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
+        .filter(([_, id, num]) => !hiddenNfts?.[`${id}_${num}`]);
```

Comment:
Is there a reason not to just do this and then remove the comment?
```suggestion
        .filter(([id, num]) => !hiddenNfts?.[`${id}_${num}`]);
```